src/pages/People.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Table } from '../components/Table';
+
+const peopleToRender = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(mom => mom.name === person.motherName);
```

Comment:
in my opinion, there is no list of MOMs or DADs, there just a list of Persons or Humans =)

btw, when i've done this task first time, i decided to use 'mom' & 'dad' the same way as you, and mentor said that's not correct